src/components/RangeSelector.js

```diff
@@ -290,12 +356,19 @@ class RangeSelector extends React.Component {
           style={styles.range}
         >
           {this.props.presets.length ? <div className='mx-rangeselector-toggle' onClick={this._handleToggleViews.bind(this)} style={styles.showPresets}>Groups</div> : null}
+	<div
+          className='mx-rangeselector-track-holder'
+          onMouseDown={this._handleClickStartOnTrack.bind(this)}
+          onMouseUp={this._handleClickOnTrack.bind(this)}
```

Comment:
I think a better name for these methods would be:
`_handleTrackMouseDown`
`_handleTrackMouseUp`